data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -162,19 +160,17 @@ backend {{ back }}
 {%             set balance_translate = {'least-connection': 'leastconn', 'round-robin': 'roundrobin', 'source-address': 'source'} %}
     balance {{ balance_translate[back_config.balance] }}
 {%         endif %}
-{# If mode is not TCP skip Forwarded #}
-{%         if back_config.mode is not vyos_defined('tcp') %}
+{# If mode is not HTTP skip X-Forwarded #}
```

Comment:
should now be `If mode is HTTP`

Reply:
I believe this comment is correct - the previous comment was not reflective of logic.
We are skipping injection of headers **unless** we using HTTP mode.

Reply:
you're right

Reply:
but would be less confusing if reworded as:
`If mode is HTTP add X-Forwarded headers`

Reply:
Agreed - I have amended commit.